jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -17,26 +17,21 @@
     </v-row>
 
     <div v-if='data_selected'>
-      <v-row>
-        <v-select
-          :items="subset_items"
-          v-model="subset_selected"
-          label="Subset"
-          hint="Select subset region for photometry."
-          persistent-hint
-        ></v-select>
-      </v-row>
+      <plugin-subset-select
+        :items="subset_items"
+        :selected.sync="subset_selected"
+        label="Aperture"
+        hint="Select aperture region for photometry."
+      />
 
       <div v-if="subset_selected">
-        <v-row>
-          <v-select
-            :items="bg_subset_items"
-            v-model="bg_subset_selected"
-            label="Subset (background)"
-            hint="Select subset region for background calculation."
-            persistent-hint
-          ></v-select>
-        </v-row>
+        <plugin-subset-select
+          :items="bg_subset_items"
+          :selected.sync="bg_subset_selected"
+          :rules="[() => bg_subset_selected!==subset_selected || 'Must not match aperture.']"

```

Comment:
I guess this is fine, though during manual testing, I did purposely set both to be same to see the residual stats after region subtracted off its own median. But I guess it doesn't have any real science use.

Reply:
I just copied the same check from the line analysis background.  If you'd prefer the check not be here, I'm happy to remove it!